firmware/src/board/ublox.cpp

```diff
@@ -620,6 +620,13 @@ bool Driver::configure(const Config& cfg, os::watchdog::Timer& wdt)
     cfg_.fix_rate_hz = std::max(cfg_.fix_rate_hz, 0.5F);
     cfg_.aux_rate_hz = std::max(cfg_.aux_rate_hz, 0.1F);
     cfg_.aux_rate_hz = std::min(cfg_.aux_rate_hz, cfg_.fix_rate_hz);
+    if (cfg_.nav_filter_model == 1) {
+        // there is no such thing as filter_model 1, don't allow it
+        cfg_.nav_filter_model = static_cast<uint32_t>(msg::CFG_NAV5::DynModel::Airborne_4g);
+    } else {
+        cfg_.nav_filter_model = std::min(cfg_.nav_filter_model, static_cast<uint32_t>(msg::CFG_NAV5::DynModel::Airborne_4g));
+        cfg_.nav_filter_model = std::max(cfg_.nav_filter_model, static_cast<uint32_t>(msg::CFG_NAV5::DynModel::Portable));
```

Comment:
I suggest to define constants next to `DynModel` constraining the range of the enumeration values, e.g. 'DynModelMin = 0', `DynModelMax = 9`. These same constants should also be used in the configuration parameter range definition. Leaving it as-is makes the code not future-proof.